watertap/costing/unit_models/dewatering.py

```diff
@@ -104,8 +104,26 @@ def cost_centrifuge(
     blk.costing_package.add_cost_factor(blk, "TIC")
     cost_blk = blk.costing_package.centrifuge
     t0 = blk.flowsheet().time.first()
+    # Assume inlet state block is `mixed_state` (IDAES Separator or WaterTAP Dewatering unit)
+    if hasattr(blk.unit_model.config, 'mixed_state_block'):
+        if blk.unit_model.config.mixed_state_block is None:
+            sb = blk.unit_model.mixed_state[t0]
+        else:
+            sb = blk.unit_model.config.mixed_state_block[t0]
+    else:
+        raise BurntToast("Costing of the dewatering unit is only compatible with an IDAES Separator or WaterTAP Dewatering Unit.")
```

Comment:
Probably not the best choice of exception to raise, but I tried to be quick about it.

Reply:
That would be better as a `TypeError` - `BurntToast` is meant as a safety for things the really should never happen.